zyte_common_items/items.py

```diff
@@ -1136,3 +1139,46 @@ class JobPosting(Item):
     metadata: Optional[JobPostingMetadata] = attrs.field(
         default=None, converter=attrs.converters.optional(MetadataCaster(JobPostingMetadata)), kw_only=True  # type: ignore
     )
+
+
+@attrs.define(kw_only=True)
+class SocialMediaPost(Item):
+    """Represents a single social media post."""
+
+    #: The url of the final response, after any redirects.
+    url: str = attrs.field(converter=url_to_str)
+
+    #: The identifier of the post.
+    postId: Optional[str] = None
+
+    #: Details of reactions to the post.
+    reactions: Optional[Reactions] = None
+
+    #: The text content of the post.
+    text: Optional[str] = None
+
+    #: The title of the post.
+    title: Optional[str] = None
```

Comment:
I don't see this attribute in https://github.com/scrapinghub/unified-schema/blob/master/unified_schema/schema/social_media_post_standard.yaml -- I'm not sure which one is correct